zipline/utils/security_list.py

```diff
@@ -102,6 +81,30 @@ def update_current(self, effective_date, symbols, change_func):
             change_func(sid)
 
 
+class SecurityListSet(object):
+
+    def __init__(self, current_date_func, lookup_func=None):
+        # provide a cut point to substitute other security
+        # list implementations.
+        self.sl_constructor = SecurityList
```

Comment:
idea is to give subclasses an override - seems confusing to put it in the arg list.